go.mod

```diff
@@ -13,7 +13,8 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
+	golang.org/x/crypto v0.0.0-20211117183948-ae814b36b871
```

Comment:
This is duplicate of #1072 , right?

Reply:
It is literally #1072! I rebased on top of it to make it easier to see compilation errors while refactoring.